cukes-sql/src/main/java/lv/ctco/cukes/sql/interceptor/InflateContextInterceptor.java

```diff
@@ -0,0 +1,59 @@
+package lv.ctco.cukes.sql.interceptor;
+
+import com.google.inject.Inject;
+import lv.ctco.cukes.core.internal.context.ContextInflater;
+import lv.ctco.cukes.core.internal.context.InflateContext;
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class InflateContextInterceptor implements MethodInterceptor {
```

Comment:
What's the difference to lv.ctco.cukes.core.internal.context.InflateContextInterceptor ?